### DISCUSSION TITLE
What happened to the issue "%LANG% Translation Progress"?

### DISCUSSION THREAD

**Discussion by lex111** [reactjs/reactjs.org-translation#235] [Team Posts]
In all repositories they are deleted now! :(

**Comment by alejandronanez** [comment]
Just to keep track of this.

**Here's Spanish issue:** https://github.com/reactjs/es.reactjs.org/issues/4

**Comment by lex111** [comment]
It seems to me that the @tesseralis account was _hacked_ and the issues were deleted,  otherwise I cannot think of any reasons for how this could happen. :(

**Comment by alejandronanez** [comment]
Oh gosh - I think that is what just happened :'(

**Comment by lex111** [comment]
That's why we all need to turn on 2FA!

**Comment by tesseralis** [comment]
I'm not sure if it was hacked, but my account did get flagged (possibly from me auto-generating so many issues?) It still visible when you're logged in?

**Comment by glaucia86** [comment]
In Portuguese our translation section was deleted it as well

reactjs/pt-BR.reactjs.org#1

we think that @tesseralis account was definitely hacked and are deleting all the translations that the Community are doing.

We have to contact UR @bvaughn @acdlite @gaearon to talk about it!!!!

**Comment by lex111** [comment]
> I'm not sure if it was hacked, but my account did get flagged (possibly from me auto-generating so many issues?) It still visible when you're logged in?

Wow! Maybe this is a GitHub bug then? Need to write to GitHub support.

UPD: After all, translations are **not** lost, then everything is not so bad...

**Comment by glaucia86** [comment]
![image](https://user-images.githubusercontent.com/1631477/52502261-7e5dad00-2bc9-11e9-822b-45fbefabd63c.png)

**Comment by lex111** [comment]
Don't panic and hope for the best :) -> https://github.com/orgs/reactjs/teams/reactjs-org-localization/discussions/11

**Comment by tesseralis** [comment]
Everything is resolved now!

**Comment by gaearon** [comment]
Lol. You're indeed too productive! :-) Glad all is good now.

**Comment by tesseralis** [comment]
Is this what it means to be a 10x engineer?

**Comment by glaucia86** [comment]
> Everything is resolved now!

Amen! :)

**Comment by alejandronanez** [comment]
Welp, all fixed!
